src/modules/workspaces/repositories/team.repository.ts

```diff
@@ -20,6 +20,10 @@ export class SequelizeWorkspaceTeamRepository {
     private teamUserModel: typeof WorkspaceTeamUserModel,
   ) {}
 
+  createTransaction(): Promise<Transaction> {
```

Comment:
We do not want to use transactions in order to scale, therefore, this function should be removed